src/config/inversion-fixes.config

```diff
@@ -2722,3 +2722,10 @@ yscec.yonsei.ac.kr
 INVERT
 iframe
 #page-sidebar
+
+================================
+
+finn.no
```

Comment:
Fix is not at the correct order, and I think you're using `dynamic` theme, so this is the wrong config.

Reply:
@Gusted Thanks for the quick reply! I have update the PR now, but was unsuccessful to view any changes when using darkreader devtools. Please let me know if I'm not using the correct syntax for making the darkreader ignore inverting my SVGs.